src/streamlink/plugins/radiko.py

```diff
@@ -49,7 +49,8 @@ def _live(self, station_id):
         return url, token
 
     def _timefree(self, station_id, start_at):
-        m3u8_url = 'https://radiko.jp/v2/api/ts/playlist.m3u8'
+        # m3u8_url = 'https://radiko.jp/v2/api/ts/playlist.m3u8' ## old url?
+        m3u8_url = 'https://tf-rpaa.smartstream.ne.jp/tf/playlist.m3u8'
```

Comment:
Don't comment out old stuff. All changes will be preserved in the git history.

Reply:
I understand. Thanks for your suggestions very much. :)